MBBSEmu/Program.cs

```diff
@@ -262,24 +264,20 @@ static void Main(string[] args)
                     return;
                 }
 
+                //Setup and Run Host
+                var host = _serviceResolver.GetService<IMbbsHost>();
+                host.Start(_moduleConfigurations);
+
                 //API Report
                 if (_doApiReport)
                 {
-                    foreach (var m in modules)
-                    {
-                        var apiReport = new ApiReport(_logger, m);
-                        apiReport.GenerateReport();
-                    }
+                    host.GenerateAPIReport();
+                    
+                    //APIREPORT done -- Stop Host
```

Comment:
nit: comments that just repeat what the code does are noise and aren't necessary

Reply:
Removed